x-pack/legacy/plugins/maps/public/layers/styles/vector/components/style_prop_editor.js

```diff
@@ -80,13 +71,6 @@ export class StylePropEditor extends Component {
   }
 
   render() {
-    const fieldMetaOptionsPopover = this.props.styleProperty.isDynamic() ? (
```

Comment:
Removing this here.  The pop-over is hardcoded for ordinal-styling.  Keeping this here on the top-level style-property wrapper was a good fit because _all_  dynamic style properties were always ordinal. Now that this is no longer the case, and categorical styling does not support any configuration params, moving this closer to its dependency rather than top-level here.